docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md

```diff
@@ -6,6 +6,9 @@ title: AVS Dashboard Onboarding
 
 This document defines interfaces that AVSs must implement for us to be able to index their data for the V1 [AVS Marketplace](https://app.eigenlayer.xyz/avs).
 
+:::info
+Your AVS will **not appear** in the AVS Marketplace unless its metadata has been assigned via the steps below.
+:::
```

Comment:
This is fine but the more important thing that we need to include is how to remove duplicates from the marketplace. If AVSs deploy a new version of their AVS, they need to queue a new tx that nil's their info uri:
```
the UI will hide AVS’s with no metadata, so we should ask the AVS to update their metadata URI to null on old versions of their deployments
```

Reply:
10-4 all done in latest commit